dim/src/core.rs

```diff
@@ -95,115 +100,308 @@ pub async fn warp_core(
+                //     routes::settings::filters::post_user_settings(conn.clone()),
+                //     routes::settings::filters::get_global_settings(conn.clone()),
+                //     routes::settings::filters::set_global_settings(conn.clone()),
+                settings::filters::get_user_settings(conn.clone())
+                    .or(routes::settings::filters::get_user_settings(conn.clone()))
+                    .or(routes::settings::filters::post_user_settings(conn.clone()))
+                    .or(routes::settings::filters::get_global_settings(conn.clone()))
+                    .or(routes::settings::filters::set_global_settings(conn.clone()))
+            }),
+        )
+        .route_service(
+            "/api/v1/host/settings",
+            warp::service({
+                //     /* settings routes */
+                //     routes::settings::filters::get_global_settings(conn.clone()),
+                //     routes::settings::filters::set_global_settings(conn.clone()),
+                routes::settings::filters::get_global_settings(conn.clone())
+                    .or(routes::settings::filters::set_global_settings(conn.clone()))
+            }),
+        )
+        .route_service(
+            "/api/v1/user/*path",
+            warp::service({
+                //     /* /api/v1/user routes */
+                //     user::filters::change_password(conn.clone()),
+                //     user::filters::delete(conn.clone()),
+                //     user::filters::change_username(conn.clone()),
+                //     user::filters::upload_avatar(conn.clone()),
+                user::filters::change_password(conn.clone())
+                    .or(user::filters::delete(conn.clone()))
+                    .or(user::filters::change_username(conn.clone()))
+                    .or(user::filters::upload_avatar(conn.clone()))
+            }),
+        )
+        .route_service(
+            "/api/v1/auth/*path",
+            warp::service({
+                //     invites::filters::get_all_invites(conn.clone()),
+                //     invites::filters::generate_invite(conn.clone()),
+                //     invites::filters::delete_token(conn.clone()),
+
+                invites::filters::get_all_invites(conn.clone())
+                    .or(invites::filters::generate_invite(conn.clone()))
+                    .or(invites::filters::delete_token(conn.clone()))
+            }),
+        )
+        //     /* static routes */
+        .route_service(
+            "/static/*path",
+            warp::service(
+                routes::statik::filters::dist_static().or(routes::statik::filters::react_routes()),
+            ),
+        )
+        .route("/ws", dim_web::axum::routing::get(ws_handler))
+        .with_state(AppState {
+            conn: conn.clone(),
+            socket_tx,
+        })
+        .layer(tower_http::trace::TraceLayer::new_for_http())
+        .layer({
```

Comment:
Don't think we need an explicit block.

Reply:
i think it's fine for now, these handers are getting tossed in the next couple of PRs anyway